modules/geo-layers/src/tile-layer/tile-layer.js

```diff
@@ -65,9 +65,9 @@ export default class TileLayer extends CompositeLayer {
         (changeFlags.updateTriggersChanged.all || changeFlags.updateTriggersChanged.getTileData));
 
     if (createTileCache) {
+      const maxZoom = this.state.maxZoom || props.maxZoom;
+      const minZoom = this.state.minZoom || props.minZoom;
```

Comment:
These will overwrite values of 0. In what circumstances is the state invalid?

Reply:
Agree with values of 0. The state is only set in MVTLayer, any other TileLayer won't have maxZoom or minZoom in the state

Reply:
Done!